src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+type Props = {
+  user: User | null
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  if (!user) {
+    return <></>;
```

Comment:
If you want to return nothing inside your component you can use just `return null;`.
`null` is valid React element.